core/src/main/java/org/elasticsearch/index/query/FuzzyQueryBuilder.java

```diff
@@ -309,6 +309,10 @@ protected void doXContent(XContentBuilder builder, Params params) throws IOExcep
                     }
                 }
             } else {
+                if (fieldName != null) {
```

Comment:
Could this be a method in `AbstractQueryBuilder`? Something like `throwParseExceptionOnMultipleFields(String queryName, String fieldName, String currentName)`?